File: app/src/main/java/com/example/calendarapp/Event.java
package com.example.calendarapp;

public class Event {

    public String title;
    public String description;
    public String location;
    //The below two vars might have to switch their datatypes
    //idk how exactly date and time should be stored
    public String time;
    public String date;

    public Event(String title, String description, String location, String time, String date){
        this.title = title;
        this.description = description;
        this.location = location;
        this.time = time;
        this.date = date;
    }

}

File: app/src/main/java/com/example/calendarapp/ui/resources/Event.kt
package com.example.calendarapp.ui.resources

import java.time.LocalDate
import java.time.LocalDateTime

class Event(
    val day: LocalDate,

    var eventName: String = "",
    var start: LocalDateTime,
    var end: LocalDateTime,

    var description: String = "",
    var clientName: String = "",
    var location: String = ""
)